services/web/server/Makefile

```diff
@@ -10,30 +10,31 @@ ROOT_DIR     = $(realpath $(CURDIR)/../../../)
 VENV_DIR    ?= $(realpath $(ROOT_DIR)/.venv)
 
 
-.PHONY: install-dev
-install-dev: ## install app in edit mode for development
+.PHONY: install
+install: ## install app in edit mode for development [DEV]
 	# installing in edit mode
 	@$(VENV_DIR)/bin/pip3 install -r requirements/dev.txt
 
+
 .PHONY: tests
-tests: ## runs tests
+tests: ## runs all tests [DEV]
```

Comment:
maybe also add a recipe for unit-test / integration-test?
it's cool for clean calls from CIs

Reply:
I thought about it but I am still playing with this concept to make it absolutely minimal and to be only used for development. Notice the flags in the tests ... e.g. pytest keeps a cache and every call starts with the previously failing tests

Reply:
did not see the -ff... nice one... anyway the --pdb you would not want inside travis... and also the -x